common/script/index.js

```diff
@@ -13,48 +15,11 @@ api = module.exports = {};
 
 api.i18n = i18n;
 
-$w = api.$w = function(s) {
-  return s.split(' ');
-};
-
-api.dotSet = function(obj, path, val) {
-  var arr;
-  arr = path.split('.');
-  return _.reduce(arr, (function(_this) {
-    return function(curr, next, index) {
-      if ((arr.length - 1) === index) {
-        curr[next] = val;
-      }
-      return curr[next] != null ? curr[next] : curr[next] = {};
-    };
-  })(this), obj);
-};
-
-api.dotGet = function(obj, path) {
-  return _.reduce(path.split('.'), ((function(_this) {
-    return function(curr, next) {
-      return curr != null ? curr[next] : void 0;
-    };
-  })(this)), obj);
-};
+import * as codeHelpers from './codeHelpers';
 
-
-/*
-  Reflists are arrays, but stored as objects. Mongoose has a helluvatime working with arrays (the main problem for our
-  syncing issues) - so the goal is to move away from arrays to objects, since mongoose can reference elements by ID
-  no problem. To maintain sorting, we use these helper functions:
- */
-
-api.refPush = function(reflist, item, prune) {
-  if (prune == null) {
-    prune = 0;
-  }
-  item.sort = _.isEmpty(reflist) ? 0 : _.max(reflist, 'sort').sort + 1;
-  if (!(item.id && !reflist[item.id])) {
-    item.id = api.uuid();
-  }
-  return reflist[item.id] = item;
-};
+$w = api.$w = codeHelpers.$w;
```

Comment:
I did a check and this is not used outside of the common file. Therefore, it does not need to be exposed on the api object.